core/trino-main/src/main/java/io/trino/sql/DynamicFilters.java

```diff
@@ -402,7 +402,7 @@ public static boolean dynamicFilter(@SqlType("T") double input, @SqlType(VARCHAR
     {
         private NullableFunction() {}
 
-        private static final String NAME = "$internal$dynamic_filter_nullable_function";
+        protected static final String NAME = "$internal$dynamic_filter_nullable_function";
```

Comment:
Why?

The class has no subclasses. Did you mean to make it package private?

Reply:
Yes, package private would work as well.